src/lint/collect-spelling-diagnostics.ts

```diff
@@ -59,7 +59,12 @@ let matchers = [
   {
     pattern: /(?<=\S)  +(?! |<\/t(d|h)>)/gu,
     message: 'multiple consecutive spaces are not allowed',
-  }
+  },
+  {
+    pattern: /[Cc]lauses? \d/gu,
+    message:
+      'clauses should be referenced using <emu-xref> tags rather than hardcoding clause numbers',
```

Comment:
should we also cover steps, not just clauses?

Reply:
[Already done](https://github.com/tc39/ecmarkup/blob/f3ade26235472c57e7ab5972dcdf0aadb31fea34/src/lint/collect-spelling-diagnostics.ts#L56).